idl/github.com/uber/cadence/history.thrift

```diff
@@ -343,6 +343,12 @@ struct QueryWorkflowResponse {
   10: optional binary queryResult
 }
 
+struct ReapplyEventsRequest {
```

Comment:
for inner history API request, plz reference start workflow execution, i.e. history service API request is a wrapper of frontend API request